### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by speyejack**
Updates the readme to direct people to the website

### CODE REVIEW

**Comment by speyejack** [comment]
Questions on more things to be removed:

- Old Install instructions
-  Game controls

Better phrasing?
Should we mention v2 coming?
Should we add a troubleshooting section pointing at website then discord?

File: README.md [hunk 1]
```diff
@@ -10,63 +10,45 @@ Welcome to the official repository for the Super Mario Odyssey Online mod! Have
 * Moon Collection is shared between all players
 * Custom Configuration Menu (Accessible by holding ZL and selecting any option in the pause/start menu)
 * Support for custom gamemodes (WIP)
-### Available Gamemodes
-* Hide and Seek
+	
 
-## SMO Version Support
+## Installation and Usage
+For the typical installation along with how to setup and use muliplayer/hide and seek, please visit the [Super Mario Odyssey website](https://smoo.it).
```
Comment (Istador) [inline README.md]: Super Mario Odyssey **_Online_** website.
Super Mario Odyssey website is a bit far stretched.

File: README.md [hunk 2]
```diff
@@ -10,63 +10,45 @@ Welcome to the official repository for the Super Mario Odyssey Online mod! Have
...
 
-```
-DEVKITPRO=/opt/devkitpro/ make
-```
-
-## Installing (Atmosphère)
-
-After a successful build, simply transfer the `atmosphere` folder located inside `starlight_patch_100` to the root of your switch's SD card.
+The [Super Mario Odyssey website](https://smoo.it) has a FAQ section that should solve many issues.
+However, for any further questions or help not covered by the site, please visit the [CraftyBoss Community Discord Server](discord.gg/jYCueK2BqD) and ask in the `help`/`help-2` channel.
```
Comment (Istador) [inline README.md:51]: Where is this Discord invite link coming from?
(Not that it expires soon or something.)

On the website I'm using https://discord.gg/w3TnB899ww which is from the video description of this YouTube video by CraftyBoss: https://www.youtube.com/watch?v=nXxnhvP_LEI

Comment (speyejack) [reply README.md:51]: That was the link in the repo description. I can switch it if you prefer

File: README.md (hunk 1 above)
Comment (speyejack) [reply README.md]: ....Good point, thank you

File: README.md (hunk 2 above)
Comment (Istador) [reply README.md:51]: No need to then, it can stay as it is.

Comment (speyejack) [reply README.md:51]: Works for me.

**Pull request merged**